services/wf/Chart.yaml

```diff
@@ -1,3 +1,3 @@
 apiVersion: v1
 name: wf
-version: 0.0.1
+version: 0.0.2
```

Comment:
I don't think you need to increment this version

Reply:
I do want to check that line in, but it's going to change soon anyway.  I'll deincrement Chart.yaml.